regression/test_invalid.py

```diff
@@ -20,7 +20,7 @@ def received_response(self, response):
 class TestInvalidResponse(functional.FunctionalTest):
 
     config = (
-        'cache 0;\n'
+        'cache 0;\nhttp_chain {\n   -> default;\n}\n'
```

Comment:
I think this should be addressed with #1650 fix

Reply:
Removed after 1650